R/epi_recipe.R

```diff
@@ -569,6 +569,10 @@ bake.epi_recipe <- function(object, new_data, ..., composition = "epi_df") {
     }
     composition <- "tibble"
   }
+
+  ## Latency checks/adjustments
+  object <- adjust_recipe_latency_before_bake(object)
```

Comment:
So because this is happening during bake, the `term_info` has the wrong column names; the specific error is
```r
Browse[1]> Error in `vctrs::vec_locate_matches()`:
! Each value of `needles` must have a match in `haystack`.
✖ Location 5 of `needles` does not have a match.
```
. Not sure if it'd be easier to adjust here, or move this inside the prep step of `step_epi_shift`

Reply:
Should be fixed by 28adec2